uiowa/uiowa
# [OnIowa Events] Map link and directions

Resolves https://github.com/uiowa/uiowa/issues/6608

# How to test

- Code review
- Test maps link
- Test directions details toggle
- Test user story "As an OnIowa site owner, I would like to output the event location information in a way that makes it easier for attendees to find the location."
```
ddev blt frontend && ddev blt ds --site=oniowa.uiowa.edu && ddev drush @oniowa.local uli  event/111/block-i-class-photo
```

Test on other sites using events:
```
ddev blt ds --site=admissions.uiowa.edu && ddev blt ds --site=iowasummerwritingfestival.uiowa.edu && ddev drush @admissions.local uli events && ddev drush @iowasummerwritingfestival.local uli events
```

Test on IWSF split layout
- Verify that fields work and appear in the same order as OnIowa
